### ISSUE TITLE
Inquiries about TFinterpy Capabilities

### ISSUE THREAD

**Issue by polxyxnn** [czwchenzhun/tfinterpy#2]
Hello,

I have several questions about the package: 
Is the package capable of dealing with large amounts of data points containing latitude and longitude with elevation data? 
Are we able to set the cell size of the interpolation, say 5 meter by 5 meter? 
Can we export the output and load it in a GIS environment?

Thank you!

**Comment by czwchenzhun** [comment]
Of course, I actually did a lot of testing using elevation data, and the size of the grid is customizable. It is actually interpolating the scatter set, and interpolating the grid is actually calculating the coordinates of all the points on the grid and passing them into the interpolator for calculation. You can save the interpolation result as a Dataframe in pandas, then export it to csv format or gslib file, and then import it to Surfer or ArcGIS and other software for use.
